packages/packages.json

```diff
@@ -0,0 +1,41 @@
+{
+    "protocol/fetchai/gym/1.0.0": "bafybeihpjm2sgnrpuwaqicikw4aybltm7xrjmf7cscpp2cy2xdoi6pekbq",
```

Comment:
We should sort the output - otherwise we might get changes in this file for no good reason

Reply:
They are already sorted but in the order of dependency tree levels